test/parallel/test-tls-invoke-queued.js

```diff
@@ -41,7 +41,7 @@ const server = tls.createServer({
       c.destroy();
     }));
     // Data on next _write() will be written but callback will not be invoked
```

Comment:
I think this comment needs updating as well?

Reply:
thanks, done!